crates/ibc/src/core/ics03_connection/events.rs

```diff
@@ -30,30 +29,25 @@ struct Attributes {
 /// is infallible, even if it is not represented in the error type.
```

Comment:
We can finally remove this doc comment now.

Reply:
Ah, missed this one.

Reply:
https://github.com/cosmos/ibc-rs/pull/208/commits/5eaa6b337e5bd47c83974268d94391ddffb66680